### PR TITLE
Staging 2.26.0 ck8s1

### PR DESCRIPTION
**Pull request by davidumea**
> [!warning]
> **This is a public repository, ensure not to disclose:**
>
> - [x] personal data beyond what is necessary for interacting with this pull request, nor
> - [x] business confidential information, such as customer names.

### What kind of PR is this?

**Required**: Mark one of the following that is applicable:

- [ ] kind/feature       
- [ ] kind/improvement   
- [ ] kind/deprecation   
- [ ] kind/documentation 
- [ ] kind/clean-up      
- [ ] kind/bug           
- [x] kind/other         

*Optional*: Mark one or more of the following that are applicable:

> [!important]
> Breaking changes should be marked `kind/admin-change` or `kind/dev-change` depending on type
> Critical security fixes should be marked with `kind/security`

- [ ] kind/admin-change 
- [ ] kind/dev-change   
- [ ] kind/security     
- [ ] [kind/adr]()      

### What does this PR do / why do we need this PR?

- Part of #394 

#### Checklist

- [ ] Proper commit message prefix on all commits
  
- Change checks:
  - [ ] The change is transparent
  - [ ] The change is disruptive
  - [ ] The change requires no migration steps
  - [ ] The change requires migration steps
- Metrics checks:
  - [ ] The metrics are still exposed and present in Grafana after the change
  - [ ] The metrics names didn't change (Grafana dashboards and Prometheus alerts are not affected)
  - [ ] The metrics names did change (Grafana dashboards and Prometheus alerts were fixed)
- Logs checks:
  - [ ] The logs do not show any errors after the change
- Pod Security Policy checks:
  - [ ] Any changed pod is covered by Pod Security Admission
  - [ ] Any changed pod is covered by Gatekeeper Pod Security Policies
  - [ ] The change does not cause any pods to be blocked by Pod Security Admission or Policies
- Network Policy checks:
  - [ ] Any changed pod is covered by Network Policies
  - [ ] The change does not cause any dropped packets in the `NetworkPolicy Dashboard`
- Audit checks:
  - [ ] The change does not cause any unnecessary Kubernetes audit events
  - [ ] The change requires changes to Kubernetes audit policy
- Falco checks:
  - [ ] The change does not cause any alerts to be generated by Falco
- Bug checks:
  - [ ] The bug fix is covered by regression tests

### CODE REVIEW

File: changelog/2.26.0.md
Comment (anders-elastisys) [inline changelog/2.26.0.md:1]: I am not sure how we usually do this, but should the changelog reflect changes from specific Kubespray commits that we have in our fork as well?

**Review by anders-elastisys — APPROVED** [review]

File: changelog/2.26.0.md
Comment (Xartos) [reply changelog/2.26.0.md:1]: There should at least be a commit that says that we upgraded to kubespray 2.26 which I'm not seeing

Comment (robinAwallace) [reply changelog/2.26.0.md:1]: I would also expect some relevant notes from upstream being here.
But then again, I see that we have not done that for other releases. 
What do you think, should we start adding more notes from upstream release notes? And maybe a link to it? @Xartos

**Review by robinAwallace — COMMENTED** [review]
The person that did the upgrade QA steps, did the updating of the terraform state work as expected?

File: changelog/2.26.0.md
Comment (davidumea) [reply changelog/2.26.0.md:1]: I added some changelog entries https://github.com/elastisys/compliantkubernetes-kubespray/pull/399/commits/31a802119a7df5baa14e9d75a674b697bec28e47

**Comment by davidumea** [comment]
> The person that did the upgrade QA steps, did the updating of the terraform state work as expected?

Yes :)

**Review by robinAwallace — APPROVED** [review]

**Review by Xartos — APPROVED** [review]

**Pull request merged**